application/atomic-appcontracts/erc20/src/main/java/net/consensys/gpact/appcontracts/atomic/erc20/soliditywrappers/GpactERC20Bridge.java

```diff
@@ -40,7 +40,7 @@
806040525081019061162a9190611b81565b61067a5760405162461bcd60e51b81526020600482015260136024820152721d1c985b9cd9995c919c9bdb4819985a5b1959606a1b6044820152606401610459565b60405163079cc67960e41b81526001600160a01b038381166004830152602482018390528416906379cc6790906044016114f8565b6000816116af5760016116b2565b60025b6001600160a01b038416600081815260036020908152604091829020849055815192835282018390529192507f75592476a227f5631d3cd8cc24fec7d4c3aec6a178f64ec2b8b91693672f613a910161123c565b60006020828403121561171857600080fd5b81356001600160e01b03198116811461173057600080fd5b9392505050565b80356001600160a01b038116811461174e57600080fd5b919050565b600080600080600060a0868803121561176b57600080fd5b61177486611737565b94506020860135935061178960408701611737565b925061179760608701611737565b949793965091946080013592915050565b6000806000606084860312156117bd57600080fd5b6117c684611737565b9250602084013591506117db60408501611737565b90509250925092565b6000602082840312156117f657600080fd5b5035919050565b6000806040838503121561181057600080fd5b8235915061182060208401611737565b90509250929050565b60008060006060848603121561183e57600080fd5b61184784611737565b925061185560208501611737565b9150604084013590509250925092565b6000806000806080858703121561187b57600080fd5b8435935061188b60208601611737565b925061189960408601611737565b9396929550929360600135925050565b80151581146118b757600080fd5b50565b600080600080608085870312156118d057600080fd5b6118d985611737565b9350602085013592506118ee60408601611737565b915060608501356118fe816118a9565b939692955090935050565b6000806040838503121561191c57600080fd5b61192583611737565b91506020830135611935816118a9565b809150509250929050565b60006020828403121561195257600080fd5b61173082611737565b60208082526010908201526f14185d5cd8589b194e881c185d5cd95960821b604082015260600190565b60208082526026908201527f4552433230204272696467653a20426c6f636b636861696e206e6f74207375706040820152651c1bdc9d195960d21b606082015260800190565b6020808252603c908201527f4552433230204272696467653a20546f6b656e206e6f74207472616e7366657260408201527f61626c6520746f2072657175657374656420626c6f636b636861696e00000000606082015260800190565b6001600160a01b039384168152919092166020820152604081019190915260600190565b8381526000602060018060a01b0385168184015260606040840152835180606085015260005b81811015611a8e57858101830151858201608001528201611a72565b81811115611aa0576000608083870101525b50601f01601f19169290920160800195945050505050565b60208082526024908201527f4552433230204272696467653a204d7573742068617665204d415050494e4720604082015263726f6c6560e01b606082015260800190565b60208082526022908201527f4552433230204272696467653a20746f6b656e206e6f7420636f6e6669677572604082015261195960f21b606082015260800190565b60208082526023908201527f4552433230204272696467653a204d75737420686176652050415553455220726040820152626f6c6560e81b606082015260800190565b600060208284031215611b9357600080fd5b8151611730816118a956fe0121836bd496e565c22481bc40c30c41db1fecf07c7376491a377f8056cfccd5a2646970667358221220ace52e9ac48b5a263fa0fa994d382dae158e0fe5e3528024b13bc2ba51d0c49464736f6c63430008090033";
```

Comment:
The wrappers appear not to be in sync with the solidity contract changes. Can you build locally and commit the latest wrappers pls?

Reply:
Hmmm.... I did a build immediately before the push.... I will do another build. I am pretty sure the meta-data at the end of the contract binary includes a time stamp. I wonder if this is what the issue is? I will investigate.